snapcraft.yaml

```diff
@@ -56,6 +56,17 @@ parts:
       - -share
       - -usr
 
+  upower-copyright:
+    plugin: nil
+    stage-packages:
+      - upower
+      - libupower-glib3
+    override-build: |
+      cp -r "${CRAFT_PART_INSTALL}/usr/share/doc" doc
+      rm -rf "${CRAFT_PART_INSTALL}"
+      install -Dm644 -t "${CRAFT_PART_INSTALL}/usr/share/doc/libupower-glib3" doc/libupower-glib3/copyright doc/libupower-glib3/changelog.*
+      install -Dm644 -t "${CRAFT_PART_INSTALL}/usr/share/doc/upower" doc/upower/copyright doc/upower/changelog.*
```

Comment:
You can use `prime:` instead to select only the bits you want, and `organize:` to change locations.

Said this, I understand that it might make sense to have a `$SNAP/doc` folder instead of the longer `$SNAP/usr/share/doc`, but we tend to keep folders of files from staged packages inside the snap (in the bases and in the system snapd), so we probably want to be consistent here.

Reply:
I have put it in `stage:`. `prime` causes conflicts.